packages/react/src/components/Tree/TreeItem.tsx

```diff
@@ -99,7 +102,32 @@ class TreeItem extends UIComponent<WithAsProp<TreeItemProps>> {
 
   static defaultProps = {
     as: 'li',
-    accessibility: defaultBehavior,
+    accessibility: treeItemBehavior,
+  }
+
+  titleRef = React.createRef<HTMLElement>()
+  treeRef = React.createRef<HTMLElement>()
+
+  actionHandlers = {
+    getFocusFromParent: e => {
+      const { open } = this.props
+      if (open) {
+        e.stopPropagation()
+        this.titleRef.current.focus()
+      }
+    },
+    setFocusToFirstChild: e => {
+      if (!open) {
```

Comment:
you forgot to read `open` from props 😄

Reply:
It's @Bugaa92 's fault. He gave me a code to copy paste at review, and he forgot to read `open` from props. Not my fault that I pasted his code mindlessly.

Will fix, thanks!